content/doc/find-help/faq.md

```diff
@@ -210,3 +210,6 @@ If a VACUUM operation needs more disk that there is remaining, migrating to the
 Clever Cloud store all your backups on [Cellar](https://www.clever-cloud.com/product/cellar-object-storage/). 
 Cellar has a replication system to protect the data stored inside. It creates three copies of your backups, each stored in a different datacenters in the PAR region.  
 This way, even if one datacenter has an accident, your backup are still safe in two different datacenters.
+
+If you have a custom backup configuration with multiple retention policies and periods, the Clever Cloud console will not display all the available backups for restoration. 
```

Comment:
This is not related to back-ups. This is more generic any custom change (conf, replica, back-ups, extensions) would be plot in the Clever Cloud console.

Not sure we should add a dedicated line for the back-ups only but something more generic maybe?

Reply:
@aurrelhebert Yes. Could you suggest a change please ? Keep in mind that we are in the back-ups section of the FAQ here. It it needs to be generic beyond back-ups, maybe a new issue should be made to address custom changes in general.

Reply:
I agree with Aurélien. This disclaimer is only valid for highly customized backups policies which doesn't concern 99% of our customers. It feels more like some information we should give to the customer alongside the setup ticket.